aries-site/src/pages/whats-new.mdx

```diff
@@ -1,17 +1,3 @@
-import { Box, ThemeContext } from 'grommet';
-
-import { ColorSwatch } from '../components';  
-import { whatsNew } from '../themes/whatsNew';
-
-<!-- NOTES for content editors modifying this file
-  - See [editing markdown instructions](https://github.com/grommet/hpe-design-system/blob/master/aries-site/README.md#editing-whats-new-page).
-  - Helpful markdown syntax reminders with this [markdown cheatsheet](https://www.markdownguide.org/basic-syntax/).
-  - Reference the "New Entry Template" provided below for the markdown used to 
-    create new "What's New" entries.
--->
-
-<ThemeContext.Extend value={whatsNew}>
-
```

Comment:
Looks like this section was accidentally removed

Reply:
Added